src/restapi/fSubscriptions.ts

```diff
@@ -16,22 +16,30 @@ export async function fOpenSubscriptionsForm(req: Request, res: Response): Promi
         res.json(newOKCallResponseWithMarkdown(msg));
         return;
     }
-
-    const form = await newSubscriptionsForm(req.body);
-    const callResponse = newFormCallResponse(form);
-    res.json(callResponse);
+    try {
+        const form = await newSubscriptionsForm(req.body);
+        res.json(newFormCallResponse(form));
+    } catch (error) {
+        res.json(newErrorCallResponseWithMessage('Unable to open subscriptions form: ' + error.message));
+    }
 }
 
 // SubmitOrUpdateSubscriptionsForm updates the subscriptions form with new values or
 // submits the form if submit button is clicked
 export async function fSubmitOrUpdateSubscriptionsForm(req: Request, res: Response): Promise<void> {
-    const form = await newSubscriptionsForm(req.body);
-    const callResponse = newFormCallResponse(form);
-    res.json(callResponse);
+    try {
+        const form = await newSubscriptionsForm(req.body);
+        res.json(newFormCallResponse(form));
+    } catch (error) {
+        res.json(newErrorCallResponseWithMessage('Unable to update subscriptions form: ' + error.message));
+    }
 }
 
 export async function fSubmitOrUpdateSubscriptionsSubmit(req: Request, res: Response): Promise<void> {
-    const app = newApp(req.body);
-    const callResponse = await app.createZDSubscription();
-    res.json(callResponse);
+    try {
+        const app = newApp(req.body);
+        res.json(await app.createZDSubscription());
```

Comment:
Same thing here with embedded await. If you prefer this syntax that it's fine with me :+1:

Reply:
I have a mix of both styles.  I'll make a choice and at least make them consistent.